ReviewComments/README.md

```diff
@@ -1,11 +1,12 @@
 # GitHub Pull Request Review Comments ![size 1.6GB](https://img.shields.io/badge/size-1.6GB-green.svg)
-The dataset was extracted from [GH Archive](https://www.gharchive.org/) and consists of:
 
-1. [25.3 million pull request review comments](https://drive.google.com/open?id=1rk6OTDrD09xVU0o_w8_dvtsLaeeUgwmP) since January 2015 till December 2018 - 1.6 GB (xz-compressed)
+[Download link.](https://drive.google.com/file/d/1bPEmq0qS_jvRlb9M1veLzyUJiAcV_Buz)
```

Comment:
Uploaded as `review_comments.csv.xz`, please copy to your gdrive and change the link